### ISSUE TITLE
Colors and what they mean

### ISSUE THREAD

**Issue by WeirDave** [alexkay/spek#73]
What do the colors mean? I have done recordings in what I think is 24/96 of a record and there are a few people that are telling me I am incorrect. This is one of the tracks I ripped form vinyl.
![24bit 96khz - kc and the sunshine band - kc and the sunshine band tk-603](https://cloud.githubusercontent.com/assets/798425/4341206/7a3f5d78-4036-11e4-9cae-bd36e55f247b.png)

Do I have an issue?

**Comment by sunk818** [comment]
Not necessarily. You have to consider the source your vinyl was pressed from. 

There's more discussion here about this:
http://www.head-fi.org/t/442888/how-to-tell-between-a-real-flac-and-a-fake-flac
